Core/HLE/proAdhoc.cpp

```diff
@@ -1290,42 +1289,47 @@ void sendChat(std::string chatString) {
 	auto n = GetI18NCategory("Networking");
 	chat.base.opcode = OPCODE_CHAT;
 	//TODO check network inited, check send success or not, chatlog.pushback error on failed send, pushback error on not connected
-	if (friendFinderRunning)
-	{
+	if (friendFinderRunning) {
 		// Send Chat to Server 
 		if (!chatString.empty()) {
 			//maximum char allowed is 64 character for compability with original server (pro.coldbird.net)
-			message = chatString.substr(0, 60); // 64 return chat variable corrupted is it out of memory?
+			std::string message = chatString.substr(0, 60); // 64 return chat variable corrupted is it out of memory?
 			strcpy(chat.message, message.c_str());
 			//Send Chat Messages
 			if (IsSocketReady(metasocket, false, true) > 0) {
 				int chatResult = send(metasocket, (const char*)&chat, sizeof(chat), MSG_NOSIGNAL);
 				NOTICE_LOG(SCENET, "Send Chat %s to Adhoc Server", chat.message);
-				name = g_Config.sNickName.c_str();
+				std::string name = g_Config.sNickName.c_str();
```

Comment:
nit: c_str() is not needed

Reply:
Ah, yeah, I was just trying to move it from a global so it wouldn't be a threading hazard.

-[Unknown]